dix/req.c

```diff
@@ -2187,16 +2221,31 @@ static int dix_cas_rops_alloc(struct m0_dix_req *req)
 	struct m0_dix_rec_op       *rec_op;
 	uint32_t                    i;
 	uint32_t                    j;
+	uint32_t                    pa_idx = 0;
+	uint32_t                    pa_nr = 0;
 	uint32_t                    max_failures;
 	struct m0_dix_cas_rop     **map = rop->dg_target_rop;
 	struct m0_dix_cas_rop      *cas_rop;
 	struct m0_dix_pg_unit      *unit;
 	bool                        del_lock;
+	uint32_t                   *skipped_sdevs = NULL;
+	uint32_t                    skipped_sdevs_num = 0;
+	uint32_t                    skipped_sdevs_max =
+		rop->dg_pver->pv_attr.pa_P;
+	uint32_t                    invalid_sdev_id = 0xFFFFFFFF;
```

Comment:
Make this a constant: `enum { INVALID_SDEV_ID = UINT32_MAX };`

Reply:
Done.